.github/workflows/build-windows.yml

```diff
@@ -0,0 +1,49 @@
+name: "Windows build"
+
+on:
+  push:
```

Comment:
Should we really build it on each push?

As I recall Geoff uses this reap to push his development branches and probably all other forks will be affected.

Reply:
Current Travis, Appveyor and Github CI build project on each push.

Reply:
Is there a reason not to?